samples/net/gsm_modem/src/main.c

```diff
@@ -93,13 +93,13 @@ static void event_handler(struct net_mgmt_event_callback *cb,
 int main(void)
 {
 	const struct device *uart_dev =
-				device_get_binding(CONFIG_MODEM_GSM_UART_NAME);
+				device_get_binding(DT_BUS_LABEL(DT_INST(0, gsm_ppp)));
```

Comment:
Better to use `DEVICE_DT_GET()` instead of looking up by label.

Reply:
Done